.github/workflows/ci.yml

```diff
@@ -35,7 +37,7 @@ env:
 concurrency:
   # Cancel all workflow runs except latest within a concurrency group. This is achieved by defining a concurrency group for the PR.
   # Non-PR builds have singleton concurrency groups.
-  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.event.number || github.sha }}
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.event.number || github.sha }}-${{ github.event_name == 'repository_dispatch' && github.event.client_payload.slash_command.args.named.sha || github.sha }}
```

Comment:
probably deserves comment that since SHA of both PR and the test-with-secrets run is same so we need something to disambiguate to prevent one from cancelling the other.

Reply:
Done!